docs/more/development-tips.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Tips for developing Yew applications
+---
+
+:::important contribute
+This document only contains information for adding supporting in IntelliJ IDEA and VSCode.
+Feel free to contribute to add instructions for your editor of choice. 
+:::
+
+## Add a template for creating components
+
+### IntelliJ IDEA
+
+1. Navigate to File | Settings | Editor | Live Templates.  
+2. Select Rust and click on + icon to add a new Live Template.
+3. Give it a name and description of your preference.
+4. Paste the following snippet in Template Text section:
+
+```rust
+use yew::prelude::*;
+
+struct $NAME$ {
+    link: ComponentLink<Self>
+}
+
+enum Msg {
+}
+
+impl Component for $NAME$ {
+    type Message = Msg;
+    type Properties = ();
+
+    fn create(_props: Self::Properties, link: ComponentLink<Self>) -> Self {
+        Self { 
+            link 
+        }
+    }
+
+    fn update(&mut self, msg: Self::Message) -> ShouldRender {
+        match msg {}
+    }
+
+    fn change(&mut self, _props: Self::Properties) -> ShouldRender {
+        false
+    }
+
+    fn view(&self) -> Html {
+        html! {
+            $HTML$
+        }
+    }
+}
+```
+
+### VSCode
+
+1. Navigate to File | Preferences | User Snippets.
```

Comment:
I think this makes things clearer?

```suggestion
1. Navigate to File > Preferences > User Snippets.
```

Reply:
IntelliJ does that so I kept it like that. I think JetBrains documentation also does it like that (not 100% sure on that though)